clevis/Sanity/simple-bind-luks/runtest.sh

```diff
@@ -50,7 +50,10 @@ rlJournalStart
         rlRun "luksmeta show -d ${lodev} -s 1" "1-100" "Check if there is no luksmeta data in slot 1"
         rlRun "wget -nv -O adv.json \"http://localhost/adv\"" 0 "Get advertisement from tang server"
 
-        rlRun  "echo -n redhat123 | clevis luks bind -f -k - -d ${lodev} tang '{ \"url\": \"http://localhost\", \"adv\": \"adv.json\" }'" 0 "clevis luks bind"
+        rlRun -s "echo -n redhat123 | clevis luks bind -f -k - -d ${lodev} tang '{ \"url\": \"http://localhost\", \"adv\": \"adv.json\" }'" 0 "clevis luks bind"
+
+        rlIsRHEL '>=8.8' && rlIsRHEL '<9' && rlAssertNotGrep "Warning" $rlRun_LOG
+        rlIsRHEL '>=9.2' && rlAssertNotGrep "Warning" $rlRun_LOG
```

Comment:
```suggestion
        rlIsRHEL '<9.2' || rlAssertNotGrep "Warning" $rlRun_LOG
```

It is better to use the "else" branch to be forward compatible